### DISCUSSION TITLE
Domanda su Amazon Ring

### DISCUSSION THREAD

**Discussion by SkunkGuru** [indomus/forum#2436] [GENERALE - Discussioni generiche]
Ho un citofono Elvox 620R che sul sito Amazon viene considerato compatibile, come modello
In realtà il citofono apre 2 diversi ingressi (cancello e portone) con 2 pulsanti diversi, e pare invece (da quel che leggo) che ring gestisca un unico ingresso.
Visto che attivo gli ingressi sempre assieme (sarebbero il 2o e 3o pulsante nero nella prima colonna), potrebbe effettivamente funzionare collegando ring ad entrambi gli ingressi in qualche modo ?

![citofono_interno](https://user-images.githubusercontent.com/15073199/232850711-7e83656a-c33e-4920-a1df-21aa61668e68.jpg)

**Comment by indomus** [comment]
A logica sì

**Reply by SkunkGuru** [reply]
per la cronaca, il ring non ha funzionato, nonostante sia dichiarato compatibile; l'assistenza non sa darmi risposte a parte "ricontrolla il cablaggio" (e tempi lunghi di risposta, 4gg di media...)
faro' il reso...

a questo punto cosa potrei usare per domotizzare la sola apertura dei 2 portoni ?

**Reply by Scialla** [reply]
un dispositivo a contatto pulito per ogni pulsante, oppure unisci insieme i due pulsanti in uno e te ne fai bastare uno. Ad esempio shelly1

**Reply by KC-inDomus** [reply]
ad esempio Sonoff RM3 pro

**Reply by SkunkGuru** [reply]
> ad esempio Sonoff RM3 pro

scusa ma non lo trovo da nessuna parte, con RM3 pro trova un broadlink ma non credo sia quello
intendi il sonoff 4CH PRO (R3)  ?

**Reply by KC-inDomus** [reply]
scusa, 4ch
https://indomus.it/componenti/itead-sonoff-4ch-4ch-pro-r3/
